jkube-kit/build/api/src/test/resources/docker/Dockerfile_expose_ports

```diff
@@ -0,0 +1,7 @@
+ARG VERSION=latest
+FROM openjdk:$VERSION
+COPY maven/target/docker-file-simple.jar /deployments/docker-file-simple.jar
+# Copying a file inside project root directory
+COPY maven/static-dir-in-project-root/my-file.txt /deployments/my-file.txt
+EXPOSE 8080
```

Comment:
Use a corner-case scenario to validate it works as expected:
- Add several EXPOSE lines (top of the document, bottom, in the middle lines).
- Add expose using all possible formats: `EXPOSE 80`, `EXPOSE 80/http`, `EXPOSE 99/udp`, `EXPOSE 8080`...
- Add duplicate EXPOSE values